docs/source/install.rst

```diff
@@ -1,9 +1,22 @@
 Install
 =======
 
+Prerequisites
+-------------
+
+UCX depends on the following system libraries being present: ``libibverbs``,
+``librdmacm``, and ``libnuma`` (``numactl`` on Enterprise Linux).  Please
```

Comment:
I think we're still missing `libibcm` here, or does one of the other libraries include that?

Reply:
Let's do that in another pass. We don't have builds with it yet.

Reply:
If we don't have builds with it yet, does that mean we won't support InfiniBand for now? That is absolutely necessary for InfiniBand and if someone tries to `enable_infiniband=True` in dask-cuda, that will fail.

Reply:
I wouldn't say "won't". It's that we "don't" have this functionality currently. We can of course update the doc when that changes.

Understood, but I think this is a different discussion.

Reply:
When the CDT is merged in we'll add the `libibcm` to the docs

Reply:
Hopefully, this happens either tomorrow or early next week

Reply:
Ok, cool. I just wanted to make sure we wouldn't miss this by accident. Thanks for clarifying.

Reply:
Documenting in PR ( https://github.com/rapidsai/ucx-py/pull/360 ). Seems like we should have the `libibcm` dependency resolved soonish.

Reply:
Awesome! Thanks @jakirkham .